backend/packages/Upgrade/src/loaders/typeormLoader.ts

```diff
@@ -54,7 +53,7 @@ export const typeormLoader: MicroframeworkLoader = async (settings: Microframewo
     name: CONNECTION_NAME.REPLICA,
     type: env.db.type as 'postgres',
     replication: {
-      master: masterHost,
+      master: exportReplicaHost,
```

Comment:
I don't think we want to add this as part of the typeorm update release. We're still using the read replica only for exports, not all db reads.

Reply:
Just to clarify once more, there is a separate postgres connection string called "export" that would call this particular connection options. And we use this connection string explicitly for only export related queries. All the other DB reads are by default calling the main DB connection option above. Still, will update this back to old where all the export related write queries are hit on `masterHost` and all export related read queries will hit the `exportReplicaHost` under the slaves mode. Any change to the `exportReplicaDBConnectionOptions` will never affect any other DB read queries apart from where its specifically targeting this 'export' connection string in the queries.

Reply:
But, yes due to your comment, I found that the mainDBConnection has slaves set to `replicaHost` which would evenly redirect reads to replica. I will update this as well. Thanks :)

Reply:
You can refer typeorm [documetation](https://orkhan.gitbook.io/typeorm/docs/multiple-data-sources#replication) for more details. Adding a small snippet out of it for a better understanding for all.

<img width="621" alt="Screenshot 2024-08-02 at 6 04 53 PM" src="https://github.com/user-attachments/assets/c91c874b-05b2-4057-aa9e-34f436538ab7">